packages/site/src/layout/AnimatedPresenceDisclosure.tsx

```diff
@@ -14,56 +15,61 @@ function AnimatedPresenceDisclosure({
   to,
 }: AnimatedPresenceDisclosureProps) {
   const [isOpen, setIsOpen] = useState(false);
+  const [isHovered, setIsHovered] = useState(false);
 
   const handleButtonClick = (e: React.MouseEvent) => {
     e.preventDefault();
     setIsOpen(!isOpen);
   };
 
+  const handleHover = (e: React.MouseEvent) => {
+    e.preventDefault();
+    setIsHovered(!isHovered);
+  };
+
   return (
-    <>
-      <Link
-        to={to ?? "/"}
-        href="https://www.google.com"
-        tabIndex={0}
-        onFocus={e => (e.currentTarget.style.boxShadow = "0 0 3px #0073e6")}
-        onBlur={e => (e.currentTarget.style.boxShadow = "none")}
-        style={{
-          display: "flex",
-          alignItems: "center",
-          justifyContent: "space-between",
-          backgroundColor: "var(--color-surface--background)",
-          outline: "none",
-          textDecoration: "none",
-        }}
-      >
-        <Typography fontWeight="semiBold" size="large" textColor="heading">
-          {title}
-        </Typography>
-        <button
-          type="button"
-          onClick={handleButtonClick}
+    <div>
+      <span className={styles.disclosureNavItem}>
+        <Link
+          to={to ?? "/"}
+          href="https://www.google.com"
+          tabIndex={0}
+          onFocus={e =>
+            (e.currentTarget.style.boxShadow = "var(--shadow-focus)")
+          }
+          onBlur={e => (e.currentTarget.style.boxShadow = "none")}
+          onMouseEnter={handleHover}
+          onMouseLeave={handleHover}
```

Comment:
I'm actually not sure if I still need these now that some of the styling lives in the CSS module